engine/crates/engine/src/registry/scalars/hex_bytes.rs

```diff
@@ -0,0 +1,53 @@
+use base64::{engine::general_purpose::STANDARD_NO_PAD, Engine as _};
+use engine_value::ConstValue;
+use serde_json::Value;
+
+use super::{DynamicParse, SDLDefinitionScalar};
+use crate::InputValueError;
+
+pub struct HexBytesScalar;
+
+impl<'a> SDLDefinitionScalar<'a> for HexBytesScalar {
+    fn name() -> Option<&'a str> {
+        Some("HexBytes")
+    }
+
+    fn description() -> Option<&'a str> {
+        Some("Bytes stored as hex. The value is returned as base64-encoded string.")
+    }
```

Comment:
Is it relevant that the bytes are stored as hex for the consumer? The input is also expected to be base64, if I read the lines below correctly.

Reply:
It looks like we already have Bytes. I'm not clear on why we need a separate type for values where the input/output format between Grafbase and the database is always a hex string. It's more connector-dependent than value dependent, postgres takes hex strings.

Reply:
This is because all translations to and from this type are common to all connectors. And all transformers are when you load a value, not when you store one. After consulting @Finistere, the decided best practice for now is to create another type, if the value is stored differently in the database. In the future, we'd like to have connector-specific transformers.